consensus/src/network.rs

```diff
@@ -348,6 +348,18 @@ impl NetworkSender {
             error!("Error broadcasting to self: {:?}", err);
         }
 
+        #[cfg(feature = "failpoints")]
+        {
+            let msg_ref = &msg;
+            fail_point!("consensus::send::broadcast_self_only", |maybe_msg_name| {
+                if let Some(msg_name) = maybe_msg_name {
+                    if msg_ref.name() != &msg_name {
+                        self.broadcast_without_self(msg_ref.clone());
+                    }
+                }
+            });
+        }
```

Comment:
Should you return at this point? Or else L363 still always runs?

Reply:
it's implicit. failpoint! macro will generate the return statement.